app/javascript/dashboard/i18n/locale/en/resetPassword.json

```diff
@@ -1,14 +1,14 @@
 {
   "RESET_PASSWORD": {
-    "TITLE": "Reset Password",
+    "TITLE": "Reset password",
     "EMAIL": {
       "LABEL": "Email",
-      "PLACEHOLDER": "Please enter your email",
-      "ERROR": "Please enter a valid email"
+      "PLACEHOLDER": "Please enter your email.",
+      "ERROR": "Please enter a valid email."
```

Comment:
I'm not sure about this change as this is a text in the input placeholder or the error text next to the input. I checked a couple of SaaS products, there doesn't seem to be a common pattern. @Hricha-Shandily Please share your thought process behind this and check if there is a recommended standard.